### ISSUE TITLE
Maximum input image resolution

### ISSUE THREAD

**Issue by AlgirdasKartavicius** [AlexeyAB/darknet#5640]
I am training yolo on my custom dataset. For training i am  using 1000x1000 images and training with parameters below:

```[net]
# Testing
#batch=1
#subdivisions=1
# Training
batch=64
subdivisions=16
width=416
height=416
channels=3
momentum=0.9
decay=0.0005
angle=0
saturation = 1.5
exposure = 1.5
hue=.1
```
Some of my dataset objects are really tiny (~10x10 px).

For inference i need to use large images, for example 4000x4000 px. With 416 width and 416 height i get really low accuracy so i tried to increase network with and height.

I can inference with parameters below (width 1248 height 1248), but when i try to increase width and height to 1664 x1664 i get **CUDA Error: out of memory**. I am using **NVIDIA Quadro P620 4GB** GPU. How can i calculate maximum width and height parameters?

```[net]
# Testing
#batch=1
#subdivisions=1
# Training
batch=64
subdivisions=16
width=1248
height=1248
channels=3
momentum=0.9
decay=0.0005
angle=0
saturation = 1.5
exposure = 1.5
hue=.1
```

**Comment by AlexeyAB** [comment]
Use subdivisions=64

**Comment by AlgirdasKartavicius** [comment]
I tried subdivisions=64, but it didn't solve my problem. I was using [Alturos.Yolo](https://github.com/AlturosDestinations/Alturos.Yolo) wrapper for inference.
I tried OpenCv dnn module and looks it works better, i can inference with higher input width and height.
So basically model size in BFLOPS should determine maximum input resolution? For example my GPU has 1.5 TFLOPS (1500 BFLOPS). 416 width and height model size is 60.1 BFLOPS.  So my maximum input resolution should be around 2000 x 2000? (sqrt(1500 / 60.1) * 416 = ~ 2000

![image](https://user-images.githubusercontent.com/26230733/82125258-5df9a680-97ad-11ea-8a4c-b4b7e9747da2.png)

**Comment by AlgirdasKartavicius** [comment]
> Use subdivisions=64
I tried subdivisions=64, but it didn't solve my problem. I was using [Alturos.Yolo](https://github.com/AlturosDestinations/Alturos.Yolo) wrapper for inference.
I tried OpenCv dnn module and looks it works better, i can inference with higher input width and height.
So basically model size in BFLOPS should determine maximum input resolution? For example my GPU has 1.5 TFLOPS (1500 BFLOPS). 416 width and height model size is 60.1 BFLOPS.  So my maximum input resolution should be around 2000 x 2000? (sqrt(1500 / 60.1) * 416 = ~ 2000

![image](https://user-images.githubusercontent.com/26230733/82125258-5df9a680-97ad-11ea-8a4c-b4b7e9747da2.png)
